docs/redirects.csv

```diff
@@ -370,3 +368,72 @@ https://community.expensify.com/discussion/5665/deep-dive-troubleshooting-expens
+https://community.expensify.com/categories/expensify-org-general-questions-and-discussions,https://www.expensify.org/
+https://community.expensify.com/discussion/4476/how-to-enable-scheduled-submit-for-a-group-policy/,https://help.expensify.com/articles/expensify-classic/reports/Automatically-submit-employee-reports
+https://community.expensify.com/discussion/4524/how-to-set-up-the-uber-integration/,https://help.expensify.com/articles/expensify-classic/connections/Uber
+https://community.expensify.com/discussion/4828/how-to-match-your-company-cards-statement-to-expensify/p1,https://help.expensify.com/articles/expensify-classic/expensify-card/Admin-Card-Settings-and-Features#reconciling-expenses-and-settlements
+https://community.expensify.com/discussion/4842/how-to-split-expenses-on-the-web-and-mobile-app,https://help.expensify.com/articles/expensify-classic/expenses/Split-an-expense
+https://community.expensify.com/discussion/4948/faq-why-didnt-my-ereceipt-generate/,https://help.expensify.com/articles/expensify-classic/workspaces/Expense-Settings#ereceipts
+https://community.expensify.com/discussion/5145/how-to-connect-to-sage-intacct-role-based-permissions-expense-reports/,https://help.expensify.com/articles/expensify-classic/connections/sage-intacct/Connect-To-Sage-Intacct
+https://community.expensify.com/discussion/5366/troubleshooting-credit-card-issues-in-expensify/,https://help.expensify.com/articles/expensify-classic/connect-credit-cards/company-cards/Troubleshooting
+https://community.expensify.com/discussion/5541/deep-dive-what-is-smartscan-and-how-it-works/,https://help.expensify.com/articles/expensify-classic/expenses/Add-an-expense#smartscan-a-receipt
+https://community.expensify.com/discussion/5678/deep-dive-secondary-login-merge-accounts-what-does-this-mean/p1,https://help.expensify.com/articles/expensify-classic/settings/account-settings/Merge-accounts
+https://community.expensify.com/discussion/5922/deep-dive-day-1-with-expensify-for-submitters/p1,https://help.expensify.com/articles/expensify-classic/getting-started/Join-your-company's-workspace
+https://community.expensify.com/discussion/8629/employee-training-e-learning-programme/p1,https://help.expensify.com/articles/expensify-classic/getting-started/Join-your-company's-workspace
+https://community.expensify.com/discussion/5124/how-to-add-your-name-and-photo-to-your-account/,https://help.expensify.com/expensify-classic/hubs/settings/account-settings
+https://community.expensify.com/post/discussion/3rd-party-integrations,https://help.expensify.com/expensify-classic/hubs/connections/
+https://community.expensify.com/post/discussion/advanced-admin-controls,https://help.expensify.com/expensify-classic/hubs/workspaces/
+https://community.expensify.com/post/discussion/becoming-an-expensifyapproved%21-accountant,https://help.expensify.com/expensify-classic/hubs/expensify-partner-program/
```

Comment:
I don't think this one is right because it has `%21` which is url encoding

Reply:
It's copied from the most commonly visited pages export from Vanilla

Reply:
the %21 is just `!`, and given nothing else is URL encoded here (ie, we have `'`, we should probably not URL encode in this file too, right? I'll add a commit.